run_tests.py

```
@@ -0,0 +1,119 @@
+        "-v",  # verbose output
+        "--tb=short",  # shorter traceback format
+        "--strict-markers",  # strict marker validation
+        "--disable-warnings",  # disable warnings
+        "tests/"  # test directory
+    ])
+
+    print(f"Running: {' '.join(cmd)}")
+    print("-" * 50)
+
+    try:
+        result = subprocess.run(cmd, cwd=Path(__file__).parent)
```

Ruff (0.13.1)

`subprocess` call: check for execution of untrusted input

(S603)